public/walkthroughs/my-custom-walkthrough/walkthrough.json

```diff
@@ -1,6 +1,36 @@
 {
   "dependencies": {
     "repos": [],
-    "services": []
+    "serviceInstances": [{
+      "metadata": { "name": "fuse" },
+      "spec": {
```

Comment:
can probably leave a similar class as the example e.g. the nodejs messaging app.